docs/maintenance.md

```diff
@@ -0,0 +1,77 @@
+# Put Graasp in maintenance mode
+
+Maintenance mode is currently setup at the infrastructure level. Go through the following steps to enable the maintenance:
+
+1. In Cloudfront, go into each distribution frontend you want to be affected
+2. Behavior > Edit the default behavior > Function association > Viewer request > Lambda@Edge > `arn:aws:lambda:us-east-1:592217263685:function:maintenance-check:2`
```

Comment:
Is there a reason why the lambda is in the `us-east-1` region ? Or am I missing something ?